### PR TITLE
Catalog layout

### PR DESCRIPTION
**Pull request by thegoldenveins**
Added Layout.js and Layout.scss files

Layout.js consists of CatalogCard.js components with different type assigned.

Difficulties for now:
-Responsivness of the catalog is absent
-Gradient hover effect as in UI kit absent

### CODE REVIEW

**Comment by thegoldenveins** [comment]
This branch includes Banner.

File: src/components/Catalog/index.scss [hunk 1]
```diff
@@ -108,4 +213,48 @@
...
+}
+//SALE CARD
+.sale-card {   
+    cursor: pointer;
+
+    &__container{
+        display: flex;
+        width: 720px;
+        height: 190px;
+    }
```
Comment (imma-fish-u) [inline src/components/Catalog/index.scss:235]: 1. check everywhere on white space line between styles (like here &__container and &__description{

File: src/components/Catalog/index.scss [hunk 2]
```diff
@@ -108,4 +213,48 @@
...
+}
+
+// curtains
+// pillows
+// lambrequins
+// blankets
+// fabrics
+// fittings
+// sale
```
Comment (imma-fish-u) [inline src/components/Catalog/index.scss]: everywhere in the end of the file add also white space line pls

**Review by imma-fish-u — CHANGES_REQUESTED** [review]
we'll be approved after small changes

**Review by imma-fish-u — APPROVED** [review]

**Pull request closed without merging**